### PR TITLE
New spec for oracle asmlib configuration

### PR DESCRIPTION
**Pull request by qinpingli**
### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [X] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This spec is used to detect the Oracle ASMlib configuration issue proactively. If the configuration doesn't ignore the single disk scan, the oracle service will crash when a single path in multipath fails. Detailed info is in the CEECBA-5575.

### CODE REVIEW

**Comment by lhuett** [comment]
@qinpingli This PR can be reviewed once the spec has been approved.

**Review by bfahr — APPROVED** [review]
Approved pending approval of the spec.

**Pull request merged**
